### DISCUSSION TITLE
Speichern von Temperatur

### DISCUSSION THREAD

**Discussion by sneakyx** [OpenCCU/OpenCCU#2498] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Speichern von Temperaturprofilen nicht möglich.
Wenn ich die bestehenden Temperaturprofile bearbeite, werden diese nicht durch "Änderungen speichern" oben rechts gespeichert. Wenn ich neu auf die Seite gehe, stehen immer noch die alten Daten dort.

### Describe the behavior you expected

Ich hatte erwartet, dass beim Klicken auf "Änderungen speichern" oben rechts die Daten zur den Heizungsventilen übertragen wird.

### Steps to reproduce the issue

1. auf Einstellungen klicken
2. "Geräte" anklicken
3. beim Heizungsventil auf "Einstellen" klicken
4. die Uhrzeiten und Temperaturen ändern
5. oben rechts auf "Änderungen speichern" klicken

### What is the version this bug report is based on?

3.71.12.20231020

### Which base platform are you running?

rpi0 (RaspberryPi)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
-
```

### Additional information

Die Umschaltung auf Urlaubs/Partymodus, manuell und automatisch funktioniert. Ich weiss auch, dass ich es einmal eingestellt habe, aber aus irgendeinem Grund funktioniert es nicht mehr.

**Comment by Baxxy13** [comment, marked as answer]
Der Button speichert z.B. Programme, Systemvariablen, Gerätenamen usw.
Geräteeinstellungen werden im Gerät selbst gespeichert. Und zwar dann wenn du innerhalb der Geräteeinstellungen unten die Buttons "OK" oder "Übernehmen" drückst. 

Ist also kein Bug sondern nur eine falsche Annahme deinerseits.

**Comment by sneakyx** [comment]
:facepalm:  Sorry, das habe ich nicht gesehen.
